H_CZ_2_2_complete_ECC_set_modified.json

```diff
@@ -1,11 +0,0 @@
-[[],
-{
-"0_5": [
-[[2,0,0,0,["da715f84238f"],[4.31710838500297678e-01,2.10647530283869677e-01]],[]]
-,[[2,0,0,2,["da715f84238f"],[4.31710838500297567e-01,2.10647530283869650e-01]],[["h", ["Q1"],["Q1"]],["h", ["Q1"],["Q1"]]]]
-,[[2,0,0,2,["da715f84238f"],[4.31710838500297622e-01,2.10647530283869677e-01]],[["h", ["Q0"],["Q0"]],["h", ["Q0"],["Q0"]]]]
-,[[2,0,0,2,["da715f84238f"],[4.31710838500297678e-01,2.10647530283869677e-01]],[["cz", ["Q0", "Q1"],["Q0", "Q1"]],["cz", ["Q0", "Q1"],["Q0", "Q1"]]]]
-,[[2,0,0,2,["da715f84238f"],[4.31710838500297678e-01,2.10647530283869677e-01]],[["cz", ["Q0", "Q1"],["Q0", "Q1"]],["cz", ["Q1", "Q0"],["Q1", "Q0"]]]]
-]
-}
-]
```

Comment:
Have you fixed the cpp code that uses this file?

Reply:
I think this file is still needed in running `running rigetti`.

Reply:
So maybe let's restore this file for now.

Reply:
I opened issue #28 to track this.

Reply:
Restored.